src/redux/auth/saga.js

```diff
@@ -14,9 +14,9 @@ function* login() {
                 method: "POST",
                 body: loginData,
             });
-            yield put(loginSuccess(response.access_token));
+            yield put(loginSuccess(response));
         } catch (error) {
-            yield put(loginFailure(error.detail ?? error));
+            yield put(loginFailure(error.detail));
```

Comment:
@ksdfg error is added here because some errors don't have a detail property. They have an error.message property, and error objects have toString overridden to be able to print a 
string message.

https://github.com/VPPR/frontend/blob/12c52a8b142dcc04640b44ca752b93f776b7843c/src/redux/auth/saga.js#L31

I had to do something like this to not use that toString() even though this is uglier than I thought. 

For trying this out, login and shut off your net. You should ideally get an error, failed to fetch, you will not get it

Reply:
hmm, I actually wanted to write a `parseError` service so we don't have to deal with the `Object [object]` errors we get from FastAPI as well. Will do in a separate PR.

Reply:
the solution for them is to do error.toString()

Reply:
Oh wait. The 422 ones?